cms/lib/xblock/tagging/test.py

```diff
@@ -152,27 +154,27 @@ def test_preview_html(self):
         self.assertEquals(div_node.get('data-runtime-version'), '1')
         self.assertIn('xblock_asides-v1', div_node.get('class'))
 
-        select_nodes = div_node.xpath('div/select')
+        select_nodes = div_node.xpath('div/div/div/div/select')
```

Comment:
This seems very fragile to me. Are there any better ways to identify the node you're looking for (perhaps by the selector or id of the `select` node?)